### ISSUE TITLE
Privacy_Preamble not localizing correctly

### ISSUE THREAD

**Issue by johngruen** [mozilla/testpilot#2219]
for some reason the privacy_preamble (See cliqz) is not getting passed in the en_us build experiments.fte file. I tried adding the field name to the utils.js file but the changes didn't get picked up. Right now we're relying on copy pasta to keep everything together.

**Comment by lmorchard** [comment]
Confused by the description of this issue, need some clarification...

> for some reason the privacy_preamble (See cliqz) is not getting passed in the en_us build experiments.fte file

Trying to understand what you're doing & not seeing. 

What command are you running to build build and what experiments.ftl file are you looking at? Did you [copy the build version from frontend/build/static/locales/en-US/experiments.ftl to locales/en-US/experiments.ftl, and include it in a pull request](https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#localization)?

> I tried adding the field name to the utils.js file but the changes didn't get picked up

Do you mean [`frontend/tasks/util.js`](https://github.com/mozilla/testpilot/blob/master/frontend/tasks/util.js)? I don't see `privacy_preamble` in there on master.

I tried adding `privacy_preamble` to the `LOCALIZABLE_FIELDS` list locally and ran `gulp content-experiments-data`. After that, the `cliqzPrivacypreamble` string showed up in `frontend/build/static/locales/en-US/experiments.ftl`. (That `content-experiments-data` task should get run as a part of the usual `npm start` in local dev or `npm static` in the production build.)

> Right now we're relying on copy pasta to keep everything together.

What copy pasta in what files? [I see `cliqzPrivacypreamble` in `locales/en-US/experiments.ftl`](https://github.com/mozilla/testpilot/blame/master/locales/en-US/experiments.ftl#L27). Also see it in the prod locale:
```
➜  testpilot git:(master) ✗ curl -s https://testpilot.firefox.com/static/locales/en-US/experiments.ftl | grep cliqzPrivacypreamble
cliqzPrivacypreamble = The Cliqz Test Pilot experiment is brought to you through collaboration between Cliqz GmbH, and Mozilla. Cliqz is designed to protect your privacy and avoid collecting sensitive information that can be used to create user profiles. However, anonymizing data like this is difficult, and it may still be possible to identify specific individuals.
```
Basically, trying to understand exactly what you did that didn't work - and what exactly the current workaround is.

**Comment by johngruen** [comment]
@lmorchard to confirm this is fixed

**Comment by lmorchard** [comment]
Yup, I think this is fixed: https://github.com/mozilla/testpilot/pull/2508#issuecomment-307211215
